packages/libs/eda/src/index.tsx

```diff
@@ -43,7 +47,8 @@ import reportWebVitals from './reportWebVitals';
 import Header from './Header';
 import MapApp from './lib/map';
 import WorkspaceApp from './lib/workspace';
-import UIThemeProvider from '@veupathdb/coreui/dist/components/theming/UIThemeProvider';
+import CoreUIThemeProvider from '@veupathdb/coreui/dist/components/theming/UIThemeProvider';
```

Comment:
Nice. 

Note to self: This is just a rename for clarity, and to disambiguate from MUI theme stuff.